go/vt/vttablet/tabletmanager/rpc_replication.go

```diff
@@ -533,6 +533,7 @@ func (agent *ActionAgent) setMasterLocked(ctx context.Context, parentAlias *topo
 	if tablet.Type == topodatapb.TabletType_MASTER {
 		tablet.Type = topodatapb.TabletType_REPLICA
 		tablet.MasterTermStartTime = nil
+		agent.setMasterTermStartTime(time.Time{})
```

Comment:
@sougou I had to add this line to get the tests to pass. Ideally we should set MasterTermStartTime on agent and push it to tablet, instead we are setting it separately on both.